gridstatus/ercot.py

```diff
@@ -3541,7 +3546,23 @@ def _handle_temperature_forecast_by_weather_zone_docs(
             # after the correction, the straight duplicate intervals remain, so we remove them
             df = df.drop_duplicates(subset=["Interval Start", "Publish Time"])
 
-        return df.sort_values("Interval Start")
+        # Raw data looks like
+        # DeliveryDate,HourEnding,Coast,East,FarWest,North,NorthCentral,SouthCentral,Southern,West,DSTFlag
+        # 11/02/2025,01:00,63.3, 58, 60, 57, 60, 63.5, 71.4, 59,N
+        # 11/02/2025,02:00,124.3, 113, 114, 111, 116.25, 120.5, 140.6, 112.8,N
```

Comment:
We did not correct the wrong values that were published last year on DST end so I have not corrected them here. However, if we did, I think dividing them in half is reasonable.